lib/vye/dgib/authentication_token_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Vye
+  module DGIB
+    class AuthenticationTokenService
+      ALGORITHM_TYPE = 'RS256'
+      E = 'AQAB'
+      TYP = 'JWT'
+      KID = 'vye'
+      USE = 'sig'
+      SIGNING_KEY = Settings.dgi.vye.jwt.private_key_path
+      RSA_PRIVATE = OpenSSL::PKey::RSA.new(File.read(SIGNING_KEY)) if File.exist?(SIGNING_KEY)
```

Comment:
I would remove the `if File.exist` here, and define an actual path to a fixture in the `Settings` config

Reply:
if statement has been removed

Reply:
It makes sense removing this causes the test to fail. It's because you need to define a fixture in the `Settings.vye.jwt.public_key_path` and the private key path as well. I would recommend just copying what's in `modules/meb_api/spec/fixtures/dgi_public_test.pem` into a `vye` fixtures directory and pointing the settings to that